core/src/test/scala/design/hamu/core/command/FibCommandSpec.scala

```diff
@@ -0,0 +1,13 @@
+package design.hamu.core.command
+
+import design.hamu.core.command.FibCommand
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class FibCommandSpec extends AnyWordSpec with Matchers {
+  "Fib command" should {
+    "generate the fibbonaci numbers" in {
```

Comment:
I'd have 2 test cases 1 for positive and 1 for negative since you have some logic with your implementation that differs on the 2.

Reply:
Also a matter of debate is maybe you can have unit tests for the other functions as well (debatable here since they are used in your main command).